docs/htmlsrc/guides/logging/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+	<head>
+			<title>Logging in Cinder</title>
+
+		<!-- keywords used for searching -->
+		<meta name="keywords" content="guide, logging, logs, log, trace, debugging">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<!-- reference to Cinder classes -->
+   		<!-- <ci seealso dox="[CLASS NAME GOES HERE]" label="[NAME OF LINK]"></ci> -->
+      	<ci seealso dox="cinder::log" label="Logging in Cinder"></ci>
+
+   		<!-- master stylesheet - these links will be replaced when compiled -->
+		<link rel="stylesheet" href="../../_assets/css/foundation.css">
+		<link rel="stylesheet" href="../../_assets/css/prism.css">
+		<link rel="stylesheet" href="../../_assets/css/style.css">
+		<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700' rel='stylesheet' type='text/css'>
+
+		<!-- Place additional stylsheet links here, which will be copied over when compiled (optional) -->
+		
+	</head>
+
+	<body id="guide-contents" class="language-c++">
+
+		<!-- CONTENT STARTS HERE -->
+		<h1>Logging in Cinder</h1>
+        
+        <p>
+        	This document provides an overview of the logging capabilities in Cinder.  You can use this, along with the sample in <em><a href="https://github.com/cinder/Cinder/tree/master/samples/Logging" target="_blank">samples/Logging</a></em>, to quickly begin utilizing Cinder's built in logging capabilities.
+        </p>
+        <a name="overview"></a>
+        <h2>Overview</h2>
```

Comment:
Each section within the page should be within <section> tags, so wherever a new h2 starts or so.